ui_framework/doc_site/src/views/code_editor/code_editor.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+
+import {
+  KuiCodeEditor
+} from '../../../../components';
+
+export default class extends Component {
+  state = {
+    value: ''
+  };
+
+  render() {
+    return (
+      <KuiCodeEditor
+        mode="less"
+        theme="github"
+        width="100%"
+        value={this.state.value}
+        onChange={(value) => this.setState({ value })}
```

Comment:
Extract a separate method for this, see https://github.com/elastic/kibana/blob/master/style_guides/react_style_guide.md#avoid-creating-a-function-and-passing-that-as-a-property-in-render-functions